gitlab/README.md

```diff
@@ -0,0 +1,44 @@
+# Nimbella Commander GitLab Command Set
+A Nimbella Commander command set for viewing GitLab project statistics, creating tickets, and assigning issues.
+
+## Features
+- View community contribution stats for GitLab projects
+- Create a ticket for a GitLab project
+- Assign issues for a GitLab project
+- View list of users in project repository
+
+## Install
+```
+/nc csm_install github:Maljean/command-sets/tree/dev/gitlab
```

Comment:
This needs to be just `/nc csm_install gitlab`. The dev command-sets are deployed to dev namespace where `csm_install gitlab` works. The above command what you typed works when as well but we have to assume we are going to be running in production.

Reply:
That's neat, thank you. I'll keep that in mind for future command sets.